tests/sps/validation/test_alternatives.py

```diff
@@ -0,0 +1,204 @@
+            <th>Anexo II Indústria</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr><td>De R$ 225.000,01 a RS 450.000,00</td>
+            <td>4,00%</td>
+            <td>4,50%</td>
+            </tr>
+            <tr>
+            <td>De R$ 450.000,01 a R$ 900.000,00</td>
+            <td>8,25%</td>
+            <td>8,00%</td>
+            </tr>
+            <tr>
+            <td>De R$ 900.000,01 a R$ 1.800.000,00</td>
+            <td>11,25%</td>
+            <td>12,25%</td>
+            </tr>
+            </tbody>
+            </table>
+            </alternatives>
+            <table-wrap-foot>
+            <fn id="TFN1">
+            <p>A informação de alíquota do anexo II é significativa</p>
+            </fn>
+            </table-wrap-foot>
+            </table-wrap>
+            </body>
+            </sub-article>
+            </article>
+            """
+        )
+        obtained = list(AlternativesValidation(self.xmltree).validation())
+        expected = []
+        self.assertListEqual(obtained, expected)
+
+    def test_validation_fail(self):
+        self.maxDiff = None
+        self.xmltree = etree.fromstring(
+            """
+            <article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" 
+            dtd-version="1.0" article-type="research-article" xml:lang="pt">
+            <body>
+            <table-wrap id="t5">
+            <label>Tabela 5</label>
+            <caption>
+            <title>Alíquota menor para prestadores</title>
+            </caption>
+            <alternatives>
+            <p>
+            altenativa
+            </p>
+            </alternatives>
+            <table-wrap-foot>
+            <fn id="TFN1">
+            <p>A informação de alíquota do anexo II é significativa</p>
+            </fn>
+            </table-wrap-foot>
+            </table-wrap>
+            </body>
+            <sub-article article-type="translation" xml:lang="en" id="TRen">
+            <body>
+            <table-wrap id="t5">
+            <label>Tabela 5</label>
+            <caption>
+            <title>Alíquota menor para prestadores</title>
+            </caption>
+            <alternatives>
+            <title />
+            <abstract />
+            </alternatives>
+            <table-wrap-foot>
+            <fn id="TFN1">
+            <p>A informação de alíquota do anexo II é significativa</p>
+            </fn>
+            </table-wrap-foot>
+            </table-wrap>
+            </body>
+            </sub-article>
+            </article>
+            """
+        )
+        obtained = list(AlternativesValidation(self.xmltree).validation())
+        expected = [
+            {
+                'title': 'Alternatives validation',
+                'parent': 'article',
+                'parent_id': None,
+                'item': 'table-wrap',
+                'sub_item': 'alternative',
+                'validation_type': 'match',
```

Comment:
@Rossi-Luciano value in list

Reply:
@Rossi-Luciano sobre fórmulas confira na documentação se é tex-math **OU** mml:math ou tex-math **E** mml:math

Reply:
OK, @robertatakenaka